pkg/cmd/gist/create/create.go

```diff
@@ -96,8 +98,24 @@ func createRun(opts *CreateOptions) error {
 		return fmt.Errorf("failed to collect files for posting: %w", err)
 	}
 
+	gistName, err := guessGistName(files)
```

Comment:
This function never raises an error. Therefore, I would think that the error return value is superfluous and can be removed